src/state-summary/state-card-toggle.html

```diff
@@ -40,5 +42,9 @@
       type: Object,
     },
   },
+
+  isHidden: function (control) {
```

Comment:
This is wrong. If we are showing a `state-card-toggle`, we should always show a toggle. When a group has control===hidden we should pick state-card-display.